test/integration/suite.spec.js

```diff
@@ -59,7 +59,7 @@ describe('suite returning a value', function () {
         done(err);
         return;
       }
-      var pattern = new RegExp('DeprecationWarning', 'g');
+      var pattern = new RegExp('deprecated', 'g');
```

Comment:
Why not use regexp literal? AFAIK this will be compiled every time as this code executed

Reply:
And this code is executed exactly once, so it doesn't matter, and I was copying the style used in the rest of this file.

Reply:
@Mingun as this code is only ran once, it is better to compile it at run-time when it is called, rather that having it loaded on script load time.